src/core/client/admin/routes/Moderate/UserHistoryDrawer/UserHistoryDrawerContainer.tsx

```diff
@@ -21,93 +23,120 @@ import styles from "./UserHistoryDrawerContainer.css";
 interface UserHistoryDrawerContainerProps {
   open: boolean;
   onClose: () => void;
-  user: UserHistoryDrawerContainer_user;
+  userID?: string;
 }
 
 const UserHistoryDrawerContainer: FunctionComponent<
   UserHistoryDrawerContainerProps
-> = ({ open, onClose, user }) => {
+> = ({ open, onClose, userID }) => {
   return (
     <Modal open={open} onClose={onClose}>
       {({ firstFocusableRef, lastFocusableRef }) => (
         <Card className={styles.root}>
-          <div className={styles.user}>
-            <Button className={styles.close} onClick={onClose}>
-              <Icon size="md">close</Icon>
-            </Button>
-            <Flex className={styles.username}>
-              <span>{user.username}</span>
-            </Flex>
-            <div className={styles.userDetails}>
-              <Flex alignItems="center" className={styles.userDetail}>
-                <Icon size="sm" className={styles.icon}>
-                  mail_outline
-                </Icon>
-                <Typography
-                  variant="bodyCopy"
-                  container="span"
-                  className={styles.userDetailValue}
-                >
-                  {user.email}
-                </Typography>
-                <CopyButton
-                  text={user.email!}
-                  variant="regular"
-                  className={styles.copy}
-                />
-              </Flex>
-              <Flex alignItems="center" className={styles.userDetail}>
-                <Icon size="sm" className={styles.icon}>
-                  date_range
-                </Icon>
-                <Typography variant="bodyCopy" container="span">
-                  {new Date(user.createdAt).toLocaleDateString("en-us", {
-                    month: "long",
-                    day: "numeric",
-                    year: "numeric",
-                  })}
-                </Typography>
-              </Flex>
-              <Flex alignItems="center" className={styles.userDetail}>
-                <Icon size="sm" className={styles.icon}>
-                  people_outline
-                </Icon>
-                <Typography
-                  variant="bodyCopy"
-                  container="span"
-                  className={styles.userDetailValue}
-                >
-                  {user.id}
-                </Typography>
-                <CopyButton
-                  text={user.id}
-                  variant="regular"
-                  className={styles.copy}
-                />
-              </Flex>
-            </div>
-            <HorizontalRule className={styles.divider} />
-          </div>
-          {open && (
-            <div className={styles.comments}>
-              <UserHistoryTabs userID={user.id} />
-            </div>
+          {open && userID && (
```

Comment:
The modal will always show the contents when the `Modal` `open` prop is true, so move this instead so that:

```tsx
<Modal open={open && userID} // ...
```

And remove the:

```tsx
{open && userID && // ...
```